cores/migrations/20240725015052_AddCategoryPrimaryKey.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add Primary Key to Categories on category_id
+ALTER TABLE categories ADD PRIMARY KEY (category_id);
+
+ALTER TABLE categories
+DROP CONSTRAINT category_id_unique CASCADE;
```

Comment:
This had a unique index but it wasn't a Primary Key, so we add a primary key and then drop the duplicate unique index/constraint